interview-app/src/main/java/ua/edu/ukma/interviewapp/InterviewApplication.java

```diff
@@ -0,0 +1,13 @@
+package ua.edu.ukma.interviewapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.
<!-- __reviewdog__:ChA0NDQ3NjRkOTgwNGUyNzlkEgpjaGVja3N0eWxl -->